components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementAdminService.java

```diff
@@ -194,8 +242,15 @@ public IdentityProvider getIdentityProvider(String federatedIdPName)
      * @throws org.wso2.carbon.identity.application.common.IdentityApplicationManagementException
      */
     public String[] getAllLocalClaimUris() throws IdentityApplicationManagementException {
-        applicationMgtService = ApplicationManagementService.getInstance();
-        return applicationMgtService.getAllLocalClaimUris(getTenantDomain());
+
+        try {
+            applicationMgtService = ApplicationManagementService.getInstance();
+            return applicationMgtService.getAllLocalClaimUris(getTenantDomain());
+        } catch (IdentityApplicationManagementException idpException) {
+            log.error("Error while retrieving all local claim URIs", idpException);
```

Comment:
since we know the tenantDomain using getTenantDomain() method. Shall we include that in the error log?